plugins/inputs/infiniband/LICENSE

```diff
@@ -0,0 +1,8 @@
+Copyright 2019 United Kingdom Research and Innovation
```

Comment:
Can you remove this file?  We don't include any additional LICENSE or copyright notices outside of the top level LICENSE.  You will still maintain copyright on this code, check the [CLA][] for details.

[CLA]: https://www.influxdata.com/legal/cla/

Reply:
Yep removed